pkg/graph/schema.graphql

```diff
@@ -54,6 +54,66 @@ type CedarSystem {
   versionId: String
 }
 
+"""
+BusinessOwnerInformation contains information about the business owner for a CEDAR system
+"""
+type CedarBusinessOwnerInformation {
+	beneficiaryAddressPurpose: [String!]
+	beneficiaryAddressPurposeOther: String
+	beneficiaryAddressSource: [String!]
+	beneficiaryAddressSourceOther: String
+	costPerYear: String
+	isCmsOwned: Boolean
+	numberOfContractorFte: String
+	numberOfFederalFte: String
+	numberOfSupportedUsersPerMonth: String
+	storesBankingData: Boolean
+	storesBeneficiaryAddress: Boolean
+}
+
+"""
+SystemMaintainerInformation contains information about the system maintainer of a CEDAR system
+"""
+type CedarSystemMaintainerInformation {
+	agileUsed: Boolean
+	businessArtifactsOnDemand: Boolean
+	deploymentFrequency: String
+	devCompletionPercent: String
+	devWorkDescription: String
+	ecapParticipation: Boolean
+	frontendAccessType: String
+	hardCodedIPAddress: Boolean
+	ip6EnabledAssetPercent: String
+	ip6TransitionPlan: String
+	ipEnabledAssetCount: Int
+	majorRefreshDate: String
+	netAccessibility: String
+	omDocumentationOnDemand: Boolean
+	plansToRetireReplace: String
+	quarterToRetireReplace: String
+	recordsManagementBucket: [String!]
```

Comment:
Same comment about array nullability as with the `CedarBusinessOwnerInformation` arrays.